presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestSqlStandardAccessControlChecks.java

```diff
@@ -38,7 +39,7 @@
     @BeforeTestWithContext
     public void setup()
     {
-        tableName = "alice_owned_table";
+        tableName = uniqueTableName();
```

Comment:
I don't understand how this helps. I assume the tests use the same TestNG execution model where the class is instantiated once, then the setup+test methods are called concurrently. If that's true, then `tableName` needs to be local to each test method.

Also, the executors will get created multiple times. They should get run once per test class instantiation.